servers/src/test/java/tachyon/master/file/meta/TTLBucketTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.file.meta;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TTLBucketTest {
+
+  @Test
+  public void equalsTest() throws Exception {
```

Comment:
Can you add another test case that also tests what happens when the bucket also has entries in mFiles?